configparser/parse_ea4tx_remotebox_config.go

```diff
@@ -24,19 +24,15 @@ func GetEA4TXRemoteboxConfig(switchName string) ([]func(*rb.Remotebox), error) {
 		return nil, fmt.Errorf("missing portname parameter for switch %s", switchName)
 	}
 
-	if !viper.IsSet(fmt.Sprintf("%s.ipaddress", switchName)) {
-		return nil, fmt.Errorf("missing ipaddress parameter for switch %s", switchName)
-	}
-
 	name := rb.Name(viper.GetString(fmt.Sprintf("%s.name", switchName)))
 	index := rb.Index(viper.GetInt(fmt.Sprintf("%s.index", switchName)))
 	portname := rb.Portname(viper.GetString(fmt.Sprintf("%s.portname", switchName)))
 	// new options for TCP connection
-	rb_ipaddress := rb.Rb_IpAddress(viper.GetString(fmt.Sprintf("%s.ipaddress", switchName)))
-	rb_ipport := rb.Rb_IpPort(viper.GetInt(fmt.Sprintf("%s.ipport", switchName)))
-	rb_Connection := rb.Rb_Connection(viper.GetInt(fmt.Sprintf("%s.Connection", switchName)))
+	Ipaddress := rb.Ipaddress(viper.GetString(fmt.Sprintf("%s.ipaddress", switchName)))
+	Ipport := rb.Ipport(viper.GetInt(fmt.Sprintf("%s.ipport", switchName)))
+	Connection := rb.Connection(viper.GetInt(fmt.Sprintf("%s.connection", switchName)))
```

Comment:
In Go, variables starting with a capital letter are public. Since these variables are private, they should start with a lower case.
However I would suggest to remove the variables `Ipaddress` and `Ipport`. See my explanation further below in the config file.